src/lib/config.js

```diff
@@ -5,6 +5,8 @@ module.exports = {
   PORT: RC.PORT,
   // ADMIN_PORT: RC.ADMIN_PORT,
   DATABASE_URI: RC.DATABASE_URI,
+  DB_CONNECTION_POOL_MIN: RC.DB_CONNECTION.DB_CONNECTION_POOL_MIN,
+  DB_CONNECTION_POOL_MAX: RC.DB_CONNECTION.DB_CONNECTION_POOL_MAX,
```

Comment:
This should be changed from `RC.DB_CONNECTION.DB_CONNECTION_POOL_MAX` to `RC.DB_CONNECTION.POOL_MAX`

Reply:
Makes logical sense - have updated.